chromeextension/background.js

```diff
@@ -1,9 +1,21 @@
 var backgroundConfig = {
     mojitoBaseUrl: 'http://localhost:8080/',
+    experimentUrl: 'http://localhost:8080/copytune/',
```

Comment:
in our case it makes sense to share the base url but we can imagine that each button may call different system. So I wouldn't share the "url" and just have it in the buttons config